test/electron/devpanel.spec.js

```diff
@@ -68,8 +68,12 @@ describe('DevTools panel for Electron', function() {
   });
 
   it('should contain INIT action', async () => {
-    const val = await this.driver.findElement(webdriver.By.xpath('//div[contains(@class, "actionListRows-")]'))
-      .getText();
+    const element = await this.driver.wait(
```

Comment:
Also changed this, let it to waiting for element visible, it seems loading time is very slow (Compared to the previous version).